src/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs

```diff
@@ -18,6 +18,12 @@ internal static class TestConfiguration
         private const string CertificatePassword = "testcertificate";
         private const string TestDataFolder = "TestData";
 
+        public const string Realm = "TEST.COREFX.NET";
+        public const string KerberosUser = "krb_user";
+        public const string Password = "password";
```

Comment:
Nit: consider adding a TODO with associated issue number here for the change to make this random